compiler/modules/nback/number.bal

```diff
@@ -12,6 +12,15 @@ final RuntimeFunction floatToIntFunction = {
     attrs: ["nounwind", "readnone", "speculatable", "willreturn"]
 };
 
+final RuntimeFunction decimalNegateFunction = {
+    name: "decimal_neg",
+    ty: {
+        returnType: LLVM_TAGGED_PTR,
+        paramTypes: [LLVM_TAGGED_PTR]
+    },
+    attrs: ["readnone"]
```

Comment:
readonly not readnone.  decimal_const is wrong too.

Reply:
Changed